Description: The tests using shell commands may time out on Docker
--- a/internal/internal_test.go	2021-05-21 09:11:21.000000000 -0700
+++ b/internal/internal_test.go	2021-08-19 12:09:06.000000000 -0700
@@ -112,7 +112,7 @@
 		t.Skip("'echo' binary not available on OS, skipping.")
 	}
 	cmd := exec.Command(echobin, "foo")
-	out, err := CombinedOutputTimeout(cmd, time.Second)
+	out, err := CombinedOutputTimeout(cmd, time.Second*10)
 
 	assert.NoError(t, err)
 	assert.Equal(t, "foo\n", string(out))
@@ -128,7 +128,7 @@
 	expected, err := cmd.CombinedOutput()
 
 	cmd2 := exec.Command(shell, "-c", "false")
-	actual, err := CombinedOutputTimeout(cmd2, time.Second)
+	actual, err := CombinedOutputTimeout(cmd2, time.Second*10)
 
 	assert.Error(t, err)
 	assert.Equal(t, expected, actual)
@@ -139,7 +139,7 @@
 		t.Skip("'sh' binary not available on OS, skipping.")
 	}
 	cmd := exec.Command(shell, "-c", "false")
-	err := RunTimeout(cmd, time.Second)
+	err := RunTimeout(cmd, time.Second*10)
 
 	assert.Error(t, err)
 }
